### PR TITLE
Feature/centres map on lads

### PR DESCRIPTION
**Pull request by patrickjfl**
The map component now zooms in on LADs when displaying data.  To achieve this we have added an LAD map bounds lookup file which contains map bounds for each LAD.  We have also added a condition to the config file which excludes county councils from the map's interactive layer.

### CODE REVIEW

File: src/ui/map/Map.svelte [hunk 1]
```diff
@@ -17,6 +19,19 @@
 
   let container;
 
+  $: {
+    if ($selectedGeography.lad) {
+      if (ladBoundsLookup[$selectedGeography.lad]) {
```
Comment (designorant) [inline src/ui/map/Map.svelte:24]: Petty but these could be joined with logical and.

**Review by designorant — CHANGES_REQUESTED** [review]
LGTM! Just one super petty bit.

One thing I've noticed that git lost track of the renamed `ladMapBoundsLookup` because it was removed and brought back in two separate commits. It's fine here but it's a good practice to do renames in one commit so git can follow the filename extension change and keep the history intact. This works as long as the changes are subtle; for bigger changes it's better to do rename as one commit, and then the content changes in another. No big deal, just something that helps when tracking changes over time.

File: src/ui/map/Map.svelte (hunk 1 above)
Comment (vmilitaru) [reply src/ui/map/Map.svelte]: Done

**Review by designorant — APPROVED** [review]
🚀

**Pull request merged**